### PR TITLE
Fix #75, Convert a Guide-Loading-Tables.txt file to markdown

### LINKED ISSUE #75: Convert a Guide-Loading-Tables.txt file to markdown
**Is your feature request related to a problem? Please describe.**
Guide-Loading-Tables.txt should be markdown and needs a scrub. 

**Describe the solution you'd like**
- Add a new file in markdown format.
- Remove old txt loading table(s) guide.

**Additional context**
Currently, we have the main guide in markdown format (see: https://github.com/nasa/cFS-GroundSystem/issues/9), so it will be consistence when we convert this "loading tables guide" to MD.

**Requester Info**
Marek Glijer
mglijer@gmail.com

### PR DESCRIPTION
**Pull request by zacnyjegomosc**
**Describe the contribution**
Fixes #75 

**Testing performed**
Steps taken to test the contribution:
1. Spellchecker.
2. Free grammar check tool for english.

**Expected behavior changes**
No impact to behavior.

**System(s) tested on**
- Hardware: MacBook Pro
- OS: macOS
- Versions: Catalina (10.15.3)
- GitHub page on Safari - Version 13.1 (15609.1.20.111.8)

**Contributor Info - All information REQUIRED for consideration of pull request**
Marek Glijer
Personal

### CODE REVIEW

File: Guide-Loading-Tables.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Loading Table(s) - Guide
+
+## Reference: cFE Application Developers Guide (cfe_tbl.dox)
+
+1. Make sure the table image you want to load is in the on-board computer. 
+1. Open Your GroundSystem. 
+    1. Select "Start Command System".
+    1. Select "Display Page" next to Table Services. This will take you to "Table Services" menu.
+    1. Select "Send" next to "Load Table".
```
Comment (skliper) [inline Guide-Loading-Tables.md:9]: Replace "Load Table" with "CFE_TBL_LOAD_CC"

File: Guide-Loading-Tables.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Loading Table(s) - Guide
+
+## Reference: cFE Application Developers Guide (cfe_tbl.dox)
```
Comment (skliper) [inline Guide-Loading-Tables.md:3]: Replace reference with deployed cFE User's Guide: https://github.com/nasa/cFS/blob/gh-pages/cFE_Users_Guide.pdf

File: Guide-Loading-Tables.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    1. Enter the location and name of the table file you want to load in the input section:
+    
+        Example: `/cf/MyTblDefault2.tbl`
+    
+    1. Select "send". If successful, you should see an event:
+    
+        `"Successful load of '/cf/<Your_Table_File_Name>' into '<Your_APP>.<Your_Table_Name>' work buffer"`
+
+    1. At this point, your table is loaded but still "inactive". Before you can "activate" the table, you will have to "validate" it. Go back to "Table Services" menu.
+    1. Select "Send" next to "Validate Table".
```
Comment (skliper) [inline Guide-Loading-Tables.md:19]: "Validate Table" = "CFE_TBL_VALIDATE_CC"

File: Guide-Loading-Tables.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        `CFE_TBL_INACTIVE_BUFFER` for ActiveTblFlag
+        `TO_LAB_APP.MyTblDefault` for TableName 
+    
+        Note - TableName is the actual table name, not the table file name. 
+    1. Select "Send". If successful, you should see an event:
+    
+        `"<Your_App_Name> validation successful for Inactive '<Your_App_Name>.<Your_Table_Name>'"`
+
+    1. Last step is to activate it. Go back to "Table Services" menu.
+    1. Select "Send" next to "Activate Table".
```
Comment (skliper) [inline Guide-Loading-Tables.md:33]: "Activate Table" = "CFE_TBL_ACTIVATE_CC"

**Comment by astrogeco** [comment]
CCB 2020-05-20 APPROVED

**Comment by astrogeco** [comment]
@skliper do we have a CLA for this?

**Comment by skliper** [comment]
@astrogeco CLA receipt confirmed.

**Pull request merged**